cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -64,6 +65,25 @@ var (
 	osutilIsMounted = osutil.IsMounted
 )
 
+// XXX: make this more flexible if there are multiple seeds on disk, i.e.
+// read boot environment in this case
```

Comment:
to be clear we cannot read the boot environment for security reasons, we would need to get it from the cmdline again. this comment needs to be adjusted